src/hyperion/experiment_plans/rotation_scan_plan.py

```diff
@@ -199,6 +200,13 @@ def rotation_scan_plan(
 
     yield from arm_zebra(zebra)
 
+    # Check topup gate
+    yield from check_topup_and_wait_if_necessary(
+        i03.synchrotron(),
+        detector_params,
+        ops_time=10.0,  # Additional time to account for rotation, is s
+    )  # TODO figure out a correct value for opts_time
```

Comment:
Can you create a corresponding github issue for this TODO and link it here? Otherwise the todo will get lost.

Reply:
Made an issue out of these comments. See #932